pyccel/ast/core.py

```diff
@@ -3184,6 +3184,9 @@ class ClassDef(ScopedAstNode):
     interfaces : iterable
         The interface methods.
 
+    doc_string : CommentBlock, optional
+        The doc string of the class.
```

Comment:
Why don't you just call it `docstring` without the underscore?

Reply:
I'm glad you said this. I kept making typos and writing `docstring`. I chose `doc_string` to match what is in `FunctionDef`. Can I go ahead and change both?

Reply:
Sure

Reply:
Done [78adce6](https://github.com/pyccel/pyccel/pull/1620/commits/78adce6893e5aa11292acee961434e370bf26e29)